weldx/asdf/tags/weldx/core/common_types.py

```diff
@@ -134,7 +134,12 @@ def from_tree(cls, tree, ctx):
 
         """
         dtype = np.dtype(tree["dtype"])
-        data = np.array(tree["data"], dtype=dtype)
+        # TODO: it would be ideal, if asdf would handle time types natively.
```

Comment:
cref https://github.com/asdf-format/asdf-standard/issues/270

Reply:
Thanks for the reference (and welcome back). Is there any roadmap for asdf standard 2.0? I guess this has to be designed super carefully in order not breaking every bodies files.

Reply:
haven't looked into the state of the asdf 2.0 standard for a while tbh